frontend/public/components/events.jsx

```diff
@@ -386,7 +386,10 @@ class EventStream extends React.Component {
       if (kind && !kindFilter(kind, obj)) {
         return false;
       }
-      if (filter && !filter.some((flt) => flt(obj.involvedObject))) {
+      if (
+        filter &&
+        !filter.some((flt) => flt({ ...obj.involvedObject, timestamp: getFirstTime(obj) }))
```

Comment:
I suggest to pass the whole event object and keep the backwards compatibility:

```suggestion
        !filter.some((flt) => flt(obj.involvedObject, obj))
```

passing just `getFirstTime` does not give us enough flexibility for the future

Reply:
done